### PR TITLE
Refactor viz.scenario

### PR DESCRIPTION
**Pull request by Zapiano**
- Rename viz.scenario to viz.scenarios
- Extract some parts of it to external functions
- Adapt to blue style guide
- Other minor refactorings

### CODE REVIEW

**Comment by ConnectedSystems** [comment]
Part of #465

File: src/viz/viz.jl [hunk 1]
```diff
@@ -12,8 +12,8 @@ for existing functions/methods.
 function explore() end
 
 # Scenario plotting methods
-function scenario() end
-function scenario!() end
+function scenarios() end
+function scenarios!() end
```
Comment (ConnectedSystems) [inline src/viz/viz.jl:16]:
```suggestion
function scenarios() end
function scenarios!() end

function scenario(args...; kwargs...)
    @warn """
    viz.scenario() is now deprecated and will be removed in ADRIA v1.0
    
    Instead, use:
        `viz.scenarios()`
    """
    
    return scenarios(args...; kwargs...)
end
```

**Review by ConnectedSystems — CHANGES_REQUESTED** [review]
Add deprecation warning and indication of usage update.

File: src/viz/viz.jl (hunk 1 above)
Comment (Zapiano) [reply src/viz/viz.jl:16]: Yes, it works.

image

**Comment by Zapiano** [comment]
@ConnectedSystems Ready for another review.

**Review by ConnectedSystems — APPROVED** [review]

**Pull request merged**